pkg/security/tests/process_test.go

```diff
@@ -139,11 +144,11 @@ func TestProcessContext(t *testing.T) {
 		},
 		{
 			ID:         "test_rule_ancestors_glob",
-			Expression: `exec.file.name == "ls" && exec.argv == "glob" && process.ancestors.file.path =~ "/usr/**"`,
+			Expression: `exec.file.name == "ls" && exec.argv == "glob" && process.ancestors.file.path in [~"/home/**", ~"/usr/**"]`,
 		},
 		{
 			ID:         "test_self_exec",
-			Expression: `exec.file.name == "syscall_tester" && exec.argv0 == "selfexec123" && process.comm == "exe"`,
+			Expression: `exec.file.name in ["syscall_tester", "exe"] && exec.argv0 == "selfexec123" && process.comm == "exe"`,
```

Comment:
Wondering why you need "exe" here ?

Reply:
In ebpfless, the reported file name of an execve of `/proc/self/exe` will be `exe`. To make this test pass as is, it would require to follow symlinks (which is not the case now). Would you prefer to skip this test instead (until we add symlink follow) ?